### PR TITLE
Multiple improvements and fixes

### PR DESCRIPTION
**Pull request by hoh**
This branch is my work in progress.

### CODE REVIEW

File: nix/service.nix [hunk 1]
```diff
@@ -4,8 +4,8 @@ let
     (pkgs.fetchFromGitHub {
       owner = "aleph-im";
       repo = "aleph-scoring";
-      rev = "hoh-add-nix";
-      sha256 = "sha256-SkLOWKql6RtdGQPIUDkqFHgCa98OzKWFL8/bQyXXHdM=";
+      rev = "b026ce63b76265ad749945a8c184ec080512c0bb";
+      sha256 = "sha256-ClwjGVnS3SEWZ3BL6M5jRwW695imEOV25PNbQOIeR8U=";
```
Comment (Psycojoker) [inline nix/service.nix]: Those changes aren't really part of this commit

File: aleph_scoring/__main__.py [hunk 2]
```diff
@@ -335,11 +335,18 @@ def export_as_html(input_file: Optional[Path]):
     os.system("jupyter nbconvert --execute Node\\ Score\\ Analysis.ipynb --to html")
 
 
-def main():
+def get_sentry_dsn(settings_):
```
Comment (Psycojoker) [inline aleph_scoring/__main__.py]: the parameter is never used because it has a "_" at the end of its name, is it wanted?

Comment (hoh) [reply aleph_scoring/__main__.py]: Nope, that's a bug.

Comment (hoh) [reply aleph_scoring/__main__.py]: Fixed, renamed as `app_settings`

**Review by Psycojoker — APPROVED** [review]
I don't have that much business knowledge here but otherwise LGTM

**Pull request merged**